python/lsst/meas/extensions/piff/piffPsfDeterminer.py

```diff
@@ -404,4 +420,22 @@ def determinePsf(
         return PiffPsf(drawSize, drawSize, piffResult), None
 
 
+def _check_stamp_size(config):
```

Comment:
Arun this is the function that ensures the stamp size is large enough when using sky coords